arlo-client/src/components/AuditForms/CalculateRiskMeasurement.test.tsx

```diff
@@ -46,10 +48,10 @@ beforeEach(() => {
     autoPrint: jest.fn(),
   } as unknown) as jsPDF
   jspdfMock.mockImplementation(() => jspdfInstance)
-  setIsLoadingMock.mockClear()
-  updateAuditMock.mockClear()
-  getStatusMock.mockClear()
-  toastSpy.mockClear()
+  sharedSetIsLoadingMock.mockClear()
+  sharedUpdateAuditMock.mockClear()
+  sharedGetStatusMock.mockClear()
+  sharedToastSpy.mockClear()
```

Comment:
Why the name change?

Reply:
The newer eslint version includes `no-shadow`, which requires that inner scopes not shadow a variable with the same name as one in an outer scope.